core/block/import/markdown/blockconverter.go

```diff
@@ -307,15 +308,14 @@ func (m *mdConverter) createFile(f *model.BlockContentFile, id string, files map
 	newFile := filepath.Join(tempDir, baseName)
 	tmpFile, err := os.Create(newFile)
 	if err != nil {
-		log.Errorf("failed to create file: %s", err.Error())
```

Comment:
`oserror.TransformError` anonymizes file path in errors from os commands

Reply:
So it should be used in logging in case we use `os` library to manipulate files.